wp-content/themes/citylimits/tribe-events/widgets/list-widget.php

```diff
@@ -78,11 +78,11 @@
 				<!-- Event Time -->
 
 				<?php do_action( 'tribe_events_list_widget_before_the_meta' ) ?>
-
+				<!--
 				<div class="tribe-event-meta">
 					<?php echo tribe_events_event_schedule_details() . ' | ' . tribe_get_venue(); ?>
 				</div>
-
+				-->
```

Comment:
If you're going to comment this section out in HTML, make sure to comment out the PHP chunk in PHP so the code doesn't run, or remove the chunk outright.